### DISCUSSION TITLE
Academy: Metamask mobile is not redirecting to site?

### DISCUSSION THREAD

**Discussion by bf-mike-mkt** [betfinio/public#18] [Bugs and Reports]
Hey I am trying to connect my wallet in academy with my phone but when I click on metamask to open in browser nothing happens. Am I doing something wrong or is this a bug?
https://github.com/user-attachments/assets/3c5451e6-1d75-40a6-85b1-39479f425155

**Comment by bf-monster** [comment]
This is issue in metamask, not betfin. Please try open academy in metamask browser and try again. Thank you
